collector/printer_object.go

```diff
@@ -341,6 +369,15 @@ func (c Collector) fetchMoonrakerPrinterObjects(klipperHost string, apiKey strin
 		customFilamentSensors[klipperHost] = *fs
 	}
 
+	mcuQuery := ""
+	for mcus := range customMicrocontrollers[klipperHost] {
```

Comment:
make this `for mcu :=` instead of `for mcus :=` as its getting a single mcu on each iteration not a collections of mcus